src/Slices/Settings/UI/Tabs/Tabs.tsx

```diff
@@ -23,7 +23,7 @@ export const Tabs: React.FC<Props> = ({ activeTab, setActiveTab }) => {
   const { authHelper, environmentHandler } = useContext(DependencyContext);
   const tokenTooltipRef = useRef<HTMLElement>();
   const selected = environmentHandler.useSelected();
-  const tokenTabDisabled = authHelper.isDisabled();
+  const tokenTabDisabled = !authHelper.getUser();
```

Comment:
This is a bit weird because now you assume that the contract of "getUser" is that authentication is disabled. This is what makes code brittle: you have implicit contracts between modules.

There are two options:
1. You make this explicit in the documentation of getUser so that if at some point we change this (maybe we can have auth without a user as some point)  it is documented
2. You add an explicit method to the interface for checking if auth is disabled.

I am a strong proponent for 2.

Reply:
I followed here logic that was in the previous authHelper. I can add explicit method, but should the logic behind it stay the same?

Reply:
+1 on option 2